### DISCUSSION TITLE
Why not commenting out first M300 for Klipper?

### DISCUSSION THREAD

**Discussion by bbinnard** [OrcaSlicer/OrcaSlicer#7441] [General]
Orca does not comment out the first M300 command when it generates GCode for a Klipper printer. This results in an "Unknown command" warning which is disturbing. Here's sample Orca GCode for a Klipper printer with additional comments by me:

; EXECUTABLE_BLOCK_START
M73 P0 R8
;TYPE:Custom
;STARTGCODE
M117 Initializing
; Set coordinate modes
G90 ; use absolute coordinates
M83 ; extruder relative mode
; Reset speed and extrusion rates
;M200 D0 ; disable volumetric E
M220 S100 ; reset speed
; Set initial warmup temps
M117 Nozzle preheat
M104 S100 ; preheat extruder to no ooze temp
M140 S60 ; set bed temp
M190 S60 ; wait for bed final temp
M300 S40 P10 ; Bip  <== Unknown command for Klipper; should be commented out
; Home
M117 Homing
G28 ; home all with default mesh bed level
; For ABL users put G29 for a leveling request
; Final warmup routine
M117 Final warmup
M104 S210 ; set extruder final temp
M109 S210 ; wait for extruder final temp
M190 S60 ; wait for bed final temp
;
 S440 P200     ; 1st beep for printer ready and allow some time to clean nozzle
;M300 S0 P250       ; wait between dual beep;  <== commented OK
;M300 S440 P200     ; 2nd beep for printer ready <== commented OK
G4 S10             ; wait to clean the nozzle
;M300 S440 P200     ; 3rd beep for ready to start printing <== commented OK

**Comment by igiannakas** [comment]
That is in your start g-code right? Orca doesnt mess with this, ideally you should make sure the commands there are valid for the printer, if the above is indeed in your start g-code configuration.

**Comment by bbinnard** [comment]
You are correct - many thanks. I'm not sure how all that code got in there - but I fixed it easily. My start g-code for Prusa is far simpler and shorter, so it didn't come from there.
